Assets/Mediapipe/Samples/Common/Scripts/ImageSource/WebCamSource.cs

```diff
@@ -17,6 +18,10 @@ namespace Mediapipe.Unity
 {
   public class WebCamSource : ImageSource
   {
+    [Header("This resolution will be matched to the nearest available")]
+    // For the default resolution, the one whose width is closest to this value will be chosen.
```

Comment:
[imo] If you use `Header` or `Tooltip`, I think you should describe what you wrote in your comment.

```suggestion
    [Tooltip("For the default resolution, the one whose width is closest to this value will be chosen")]
```